src/DetectChanges/BCBreak/FunctionBased/ParameterByReferenceChanged.php

```diff
@@ -30,9 +30,9 @@ public function __construct()
 
     public function __invoke(ReflectionFunctionAbstract $fromFunction, ReflectionFunctionAbstract $toFunction) : Changes
     {
-        /** @var ReflectionParameter[] $fromParameters */
+        /** @var array<int, ReflectionParameter> $fromParameters */
```

Comment:
As discussed elsewhere (I think it was BetterReflection), this currently clashes with JetBrains tooling, and the `array_values()` already implies the `int[]` keys

Reply:
> currently clashes with JetBrains tooling

uggghhh - ok, will change

Reply:
Yes, this is [7 years old issue](https://youtrack.jetbrains.com/issue/WI-6558) that is still not addressed.
Right now Jetbrains guys are [claiming to be waiting for PSR-5](https://twitter.com/neuro159/status/1029335046325514240), which doesn't really fix anything (no proper support for generics - only collections, no intersections, ...) and doesn't even evolve. Based on that, I don't expect any progress or useful outcome in the near future.

Reply:
Looks like at least PSR-5 is going that direction (for arrays)